src/project/projecttypes.h

```diff
@@ -164,6 +164,24 @@ struct ProjectMeta
     {
         return io::filename(filePath, includingExtension);
     }
+
+    bool operator==(const ProjectMeta other) const
+    {
+        if (filePath != other.filePath || title != other.title || subtitle != other.subtitle || composer != other.composer
```

Comment:
This implementation will be very difficult to debug. I would suggest rewriting it as follows:

```
bool equal = filePath == other.filePath;
equal &= title == other.title;
equal &= sutitle == other.subtitle;
....

return equal;
```

Yes, there will be more code, but it will be much easier to debug

Reply:
Okay that's a better way